source/developers/references/tools_for_maintainers.rst

```diff
@@ -0,0 +1,30 @@
+Tools for Maintainers
+#####################
+
+.. list-table::
+
+   * - Tool
+     - What to Use it For
+   * - `Slack Channel`_
+     - - Coordinating with other maintainers.
+       - Sharing experiences as a maintainer.
+   * - `Backstage`_
+     - - Seeing what you own.
+       - Seeing who ows what.
+       - Seeing how repos relate to each other.
+   * - `Grimoire`_
+     - - Monitor your personal PR Backlog.
+       - Monitor the PR backlog in repos you own.
+       - Make graphs of various github activity.
+   * - `Discourse`_
+     - - Discussing complex issues with the community.
```

Comment:
Conversations that will be valuable to other community members in the future, like how to fix some non-unique problem?